internal/pkg/config/profiles_test.go

```diff
@@ -182,3 +183,87 @@ func TestImportProfile(t *testing.T) {
 		})
 	}
 }
+
+func TestExportProfile(t *testing.T) {
+	// Create directory where the export configs should be stored
+	testDir, err := os.MkdirTemp(os.TempDir(), "stackit-cli-test")
+	if err != nil {
+		t.Fatal(err)
+	}
+	t.Cleanup(func() {
+		func(path string) {
+			err := os.RemoveAll(path)
+			if err != nil {
+				fmt.Printf("could not clean up temp dir: %v\n", err)
+			}
+		}(testDir)
+	})
+
+	defaultConfigFolderPath = filepath.Join(testDir, "config")
+	err = os.Mkdir(defaultConfigFolderPath, 0o750)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Create prerequisite profile
+	p := print.NewPrinter()
+	profileName := "export-profile-test"
+	err = CreateProfile(p, profileName, true, true)
+	if err != nil {
+		t.Fatalf("could not create prerequisite profile, %v", err)
+	}
+	initConfig(defaultConfigFolderPath)
```

Comment:
```suggestion
	testConfigFolderPath = filepath.Join(testDir, "config")
	err = os.Mkdir(defaultConfigFolderPath, 0o750)
	if err != nil {
		t.Fatal(err)
	}

	// Create prerequisite profile
	p := print.NewPrinter()
	profileName := "export-profile-test"
	err = CreateProfile(p, profileName, true, true)
	if err != nil {
		t.Fatalf("could not create prerequisite profile, %v", err)
	}
	initConfig(testConfigFolderPath)
```

Reply:
I set here the global variable `globalConfigFolderPath` from `config.go. This is used in the `CreateProfile()`function, to get the profile folder path by calling `GetProfileFolderPath()`

Reply:
Don't you pass it to the `initConfig` function as argument? And only there the global variable is initialized, right?

I think currently you overwrite the global `defaultConfigFolderPath` in the test, then call `initConfig` with the new value (where it's overwritten again)

Reply:
Yes but I need to set the `defaultConfigFolderPath` before the call of the `CreateProfile()`. Because otherwise, it would create a profile folder in the normal path and not in the temp directory which I create for the test. And after the call of the `CreateProfile()`, i need to call the `initConfig()` and `Write()`. This will create the `cli-config.json` in the profile folder.

To avoid the duplicated set of `defaultConfigFolderPath`, I could remove the initialization of it in the `initConfig()` and add it to the `InitConfig()` before it calls `initConfig()`

Reply:
So this would avoid the duplicate initialization
```
func InitConfig() {
	defaultConfigFolderPath = getInitialConfigDir()
	initConfig()
}

func initConfig() {
	profileFilePath = getInitialProfileFilePath() // Profile file path is in the default config folder
        ...
}

```